x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('Policy Details', () => {
   describe('when displayed with invalid id', () => {
     let releaseApiFailure: () => void;
     beforeEach(() => {
-      http.get.mockImplementationOnce(async () => {
+      http.get.mockImplementation(async () => {
```

Comment:
since sendGetEndpointSecurityPackage now gets called, this is no longer the first